src/main/java/com/faforever/client/preferences/Preferences.java

```diff
@@ -71,6 +73,10 @@ public WindowPrefs getMainWindow() {
     return mainWindow;
   }
 
+  public LanguagePrefs getLanguagePrefs() {
```

Comment:
`getLanguage()`, but I know that we have the duplicated name then.
Therefore I suggest `LocalizationPrefs getLocalization()`